### PR TITLE
fix: action structure should be in payload

### PR DESCRIPTION
**Pull request by jmfrancois**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

all action props are spread on the button tag... but type exists on a button.
So we have warning to said button type is invalid.
This also has an other problem we don't know what to dispatch so we also dispatch stuff like icon, ... button props in redux.

**What is the new behavior?**

put all action data in the payload attribute

action has special attributes
* actionCreator
* model
* payload (new, has the type for redux, and all static stuff)

**Does this PR introduce a breaking change?**

- [x] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration
path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://135.talend.surge.sh)

File: packages/cmf/__tests__/action.test.js [hunk 1]
```diff
@@ -76,9 +76,9 @@ describe('CMF action', () => {
 		};
 		const action = actionAPI.getActionObject(context, id, event, data);
 		expect(typeof action).toBe('object');
-		expect(action.id).toBe('menu:article');
-		expect(action.name).toBe('My article');
-		expect(action.icon).toBe('icon-article');
+		expect(action.id).toBe();
```
Comment (acateland) [inline packages/cmf/__tests__/action.test.js]: ? to Be what ?

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://135.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://135.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://135.talend.surge.sh)

**Pull request merged**